### PR TITLE
Ui

### PR DESCRIPTION
**Pull request by ekumamait**
This PR creates the User Interface for FastFOODFast and closes 
#160536947
#160483982
#160484037
#160484199
#160484402
#160484459
while Responsiveness still pending.
So kindly view the demo on a large screen and give thoughts.  
Thanks

### CODE REVIEW

**Comment by walimike** [comment]
This is good, I actually find your UI responsive so thumbs up, please also review mine here; https://github.com/walimike/fast-food-fast/pull/1

**Comment by Luleherll** [comment]
its fine. i like the hover dropdown

**Comment by Phionamary** [comment]
@ekumamait
Really good work on on your UI so far 👍
Imma need some pointers  from what i see..

**Review by okellogabrielinnocent — APPROVED** [review]
Nice work here

File: UI/dash.html [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+            <div style="cursor:pointer;" class="menu_item">
+                <div class="snack-data">
+                    <small>KUNGPAO CHICKEN</small>
+                </div>
+                <h2 class="snack-title">2pc Kungpao</h2>
+                <img src="img/kungpao.JPG">
+                <p class="snack-content">2pc deep fried kungpao chicken with 
+                    a filling pack of fries and free drink</p>
+            </div>
+            <div style="cursor:pointer;" class="menu_item">
```
Comment (kyakusahmed) [inline UI/dash.html:111]: good work but i would suggest you try to keep all your style within the .css files

**Review by kyakusahmed — CHANGES_REQUESTED** [review]

File: UI/dash.html (hunk 1 above)
Comment (kdaglas) [reply UI/dash.html:111]: yea am with @kyakusahmed keep it in the external css... if they are bigger project there might be a possibility of confusion... the rest is really great

**Comment by ekumamait** [comment]
Thanks guys let me work on that

**Pull request closed without merging**